# HG changeset patch
# User John W. Eaton <jwe@octave.org>
# Date 1219169954 14400
#      Tue Aug 19 14:19:14 2008 -0400
# Branch release-3-0-x
# Node ID 22876f8e0f1790aeefee069cec99428a721a97bb
# Parent  1695e4627d2b2c5ba56dfee02d05f923d104263d
fix typo in last commit

diff --git a/src/pt-assign.cc b/src/pt-assign.cc
--- a/src/pt-assign.cc
+++ b/src/pt-assign.cc
@@ -155,17 +155,17 @@ maybe_warn_former_built_in_variable (con
     warning_with_id ("Octave:built-in-variable-assignment",
 		     "%s is no longer a built-in variable; please read the NEWS file or type `news' for details",
 		     nm.c_str ());
 }
 
 tree_simple_assignment::tree_simple_assignment
   (tree_expression *le, tree_expression *re,
    bool plhs, int l, int c, octave_value::assign_op t)
-    : tree_expression (l, c), lhs (le), rhs (re), preserve (plhs), etype (t)
+    : tree_expression (l, c), lhs (le), rhs (re), preserve (plhs), etype (t),
       first_execution (true) { }
 
 tree_simple_assignment::~tree_simple_assignment (void)
 {
   if (! preserve)
     delete lhs;
 
   delete rhs;
@@ -302,17 +302,17 @@ tree_simple_assignment::accept (tree_wal
   tw.visit_simple_assignment (*this);
 }
 
 // Multi-valued assignment expressions.
 
 tree_multi_assignment::tree_multi_assignment
   (tree_argument_list *lst, tree_expression *r,
    bool plhs, int l, int c, octave_value::assign_op t)
-    : tree_expression (l, c), lhs (lst), rhs (r), preserve (plhs), etype (t)
+    : tree_expression (l, c), lhs (lst), rhs (r), preserve (plhs), etype (t),
       first_execution (true) { }
 
 tree_multi_assignment::~tree_multi_assignment (void)
 {
   if (! preserve)
     delete lhs;
 
   delete rhs;
